pallets/gear/src/tests.rs

```diff
@@ -1662,6 +1662,54 @@ fn exit_handle() {
     })
 }
 
+#[test]
+fn set_code() {
```

Comment:
set_core should be tested, not the fact that you set something to 0 and check if it is 0